tests/Application/config/sylius/1.9/packages/security.yaml

```diff
@@ -1,9 +1,13 @@
 parameters:
```

Comment:
Please align with PluginSkeleton by having this configuration file also for 1.8.

Reply:
I copy/pasted the files directly from the 1.10 tagged release without any changes.

Reply:
Ok sorry this was my mistake.